packages/extension-chakra-storefront/src/components/with-layout/with-layout.tsx

```diff
@@ -334,6 +336,7 @@ const withLayout = <P extends object>(WrappedComponent: React.ComponentType<P>)
                         {!isCheckout ? <Footer /> : <CheckoutFooter />}
 
                         <AuthModal {...(authModal as any)} />
+                        <DntNotification {...dntNotification} />
```

Comment:
Previously, the DNT banner was not showing up. It's because we hadn't inserted this component.